Framework/MDAlgorithms/test/LoadGaussCubeTest.h

```diff
@@ -0,0 +1,61 @@
+// Mantid Repository : https://github.com/mantidproject/mantid
+//
+// Copyright &copy; 2023 ISIS Rutherford Appleton Laboratory UKRI,
+//   NScD Oak Ridge National Laboratory, European Spallation Source,
+//   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+// SPDX - License - Identifier: GPL - 3.0 +
+#pragma once
+
+#include <cxxtest/TestSuite.h>
+
+#include "MantidAPI/AnalysisDataService.h"
+#include "MantidDataObjects/MDHistoWorkspace.h"
+#include "MantidMDAlgorithms/LoadGaussCube.h"
+
+using namespace Mantid::DataObjects;
+
+using Mantid::MDAlgorithms::LoadGaussCube;
+
+class LoadGaussCubeTest : public CxxTest::TestSuite {
+public:
+  static LoadGaussCubeTest *createSuite() { return new LoadGaussCubeTest(); }
+  static void destroySuite(LoadGaussCubeTest *suite) { delete suite; }
+
+  void test_Init() {
+    LoadGaussCube alg;
+    TS_ASSERT_THROWS_NOTHING(alg.initialize())
+    TS_ASSERT(alg.isInitialized())
+  }
+
```

Comment:
can we have a test to check the validation?

Reply:
Yep good shout!